lua/nvim-web-devicons/icons-default.lua

```diff
@@ -710,6 +722,12 @@ local icons_by_file_extension = {
     cterm_color = "214",
     name = "7z",
   },
+  ["a"] = {
```

Comment:
This and `.so` are not linux or unix specific and will look strange to windows programmers.

How about we use nf-cod-library ?

Reply:
Looks fine, didn't know, 8df461d5f05d